node/rpc/src/mpool_api.rs

```diff
@@ -190,7 +190,7 @@ where
     }
     let nonce = data.mpool.get_sequence(&from).await?;
     umsg.sequence = nonce;
-    let key = wallet::find_key(&key_addr, &*keystore)?;
+    let key = wallet::Key::try_from(wallet::try_find(&key_addr, &mut *keystore)?)?;
```

Comment:
If you think `&mut *` looks unclean, there's `deref_mut` (`keystore.deref_mut()`) method as well (but it needs an import of `std::ops::DerefMut`).